### PR TITLE
[14.0] [MIG] auth_user_case_insensitive: Module migrated in v14.

### PR DESCRIPTION
**Pull request by Chandresh-OSI**

### CODE REVIEW

File: auth_user_case_insensitive/__init__.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # Copyright 2015-2017 LasLabs Inc.
+# Copyright 2021 Open Source Integrators
```
Comment (dreispt) [inline auth_user_case_insensitive/__init__.py:2]: This is best removed: porting a module does is not considered original work, and does not grant copyrights.

**Review by dreispt — APPROVED** [review]
Minor fixes, but LGTM

**Review by primes2h — APPROVED** [review]
LGTM

**Pull request merged**